CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(PracticalToolsForSimpleDesign)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
+# set(CMAKE_COLOR_DIAGNOSTICS ON)
```

Comment:
not necessary, don't touch @NOOBDY 's sacred `CMakeLists.txt`

Reply:
wdym? Removing the line even if it's comment?

Reply:
i mean i don't have anything to say